AEFeinstein/Super-2024-Swadge-FW
# MacOS Documentaion Update

### Description

- Added 'cmake' to the install list 
- Added the 'download from git and install' section from the linux section
- Changed 'to stop pulseaudio, use `brew services start pulseaudio` to `brew services stop pulseaudio'
- Added not about COM, /dev/tty.* and /dev/cu.* for futures devs.

### Test Instructions

On a clean Mac:
- Create a new esp folder where it will not interfere with any current installs
- Follow new installation instructions
- Verify that the base Swadge 2024 project compiles 

### Ticket Links

https://github.com/AEFeinstein/Super-2024-Swadge-FW/issues/204

### Readiness Checklist
